src/coreclr/tools/Common/TypeSystem/Common/Utilities/CustomAttributeTypeNameParser.cs

```diff
@@ -191,7 +191,8 @@ private TypeDesc GetSimpleTypeFromModule(TypeName typeName, ModuleDesc module)
                         return canonType;
                 }
 
-                (string typeNamespace, string name) = TypeNameHelpers.Split(fullName);
+                string typeNamespace = typeName.Namespace;
+                string name = typeName.Name;
```

Comment:
Declaring local variables seems unnecessary, inline the value into the uses below.